dump/dump-to-bucket.py

```diff
@@ -176,7 +176,8 @@ def flatten_tree_to_ident_hashes(item_or_tree):
 assert list(flatten_tree_to_ident_hashes(test_tree)) == test_tree_idents
 
 
-def scrape_version(id, version, host, visited_locs, book=None):
+def scrape_version(id, version, host, visited_locs, book=None,
+                   composite_page=False):
```

Comment:
`is_<thing>` is usually a better naming for flags. Citation is some part of Expert Python Programming.

But this is fine. Just mentioning it. 😸

Reply:
You're right, let me change it to `is_composite_page`, I did think it was a bit strange :sweat_smile: